app.js

```diff
@@ -34,6 +34,7 @@ app.use(helmet({
 }));
 app.use(bodyParser.json());
 app.use('/client', express.static(path.join(__dirname, 'dist')));
+app.get('/Coral.js', (req, res) => res.sendFile(path.join(__dirname, 'client/coral-embed-api/index.js')));
```

Comment:
Please update the route to `/coral-latest.js`. This will allow us to potentially host multiple versions in a semantic way.